install/install.go

```diff
@@ -49,35 +61,101 @@ type Options struct {
 	Local           bool
 	Save            bool
 	Verify          bool
-	Env             envOption
+	Legacy          bool
 }
 
-type envOption struct {
-	Auto        bool
-	Environment string
-}
+func getOperatorCR(o Options) *Operatorv1.KubeArmorConfig {
+	ns := o.Namespace
+	if o.Tag != "" {
+		o.KubearmorImage = updateImageTag(o.KubearmorImage, o.Tag)
+		o.InitImage = updateImageTag(o.InitImage, o.Tag)
+		o.ControllerImage = updateImageTag(o.ControllerImage, o.Tag)
+		o.RelayImage = updateImageTag(o.RelayImage, o.Tag)
+	}
```

Comment:
Someone might want to specify a different tag for only one of the images but same for all other.
For example, if I run
```
$ ./karmor install --tag v1.0 --relay-image=kubearmor/kubearmor-relay-server:stable --save
```
the generated KubeArmorConfig CR should have `v1.0` tag for all the images but `kubearmor/kubearmor-relay-server:stable` for `kubearmorRelayImage`.
If no tag is given in the specified image, for e.g. someone specified only `kubearmor/kubearmor-relay-server` take it from `--tag`.